frontend/src/components/Dashboard.vue

```diff
@@ -45,33 +48,37 @@
               <template v-slot:activator="{ on }">
                 <v-icon v-on="on" small>mdi-help-circle</v-icon>
               </template>
-             <p class="tooltip">Total Unique User Count by IDP + MHSU Realms that do not use an IDP</p>
+              <p class="tooltip">
+                Total Unique User Count by IDP + MHSU Realms that do not use an
+                IDP
+              </p>
             </v-tooltip>
           </div>
           <p class="single-stat">{{ totalNumberOfUsers }}</p>
           <v-skeleton-loader
             v-if="totalNumberOfUsersLoadingStatus"
             ref="dashboardSkeleton"
             type="text"
-            max-width="60px">
+            max-width="60px"
+          >
```

Comment:
I assume this accidental, but just a reminder to avoid including formatting changes in PR. If you want to do formatting, open a second PR later.